deployer/src/config_manager.py

```diff
@@ -168,10 +168,6 @@ def update_config(self, config_name):
                 emails.add(config_name, self.private_dir)
 
         def remove_config(self, config_name):
-            config_id = str(self.inverted_actual_configs[config_name])
```

Comment:
(the remove_config method is not removing config anymore; so what does it actually do?

Reply:
It delete the keys and the index. Still required